src/components/document/editable-document-content.tsx

```diff
@@ -13,14 +13,25 @@ import { ProblemDocument } from "../../models/document/document-types";
 import { WorkspaceMode } from "../../models/stores/workspace";
 
 import "./editable-document-content.scss";
+import { getToolContentInfoByTool } from "../../models/tools/tool-content-info";
+import { IToolButtonConfig } from "../tool-button";
```

Comment:
Conventionally, the `.scss` import should be last.

Reply:
done